### DISCUSSION TITLE
errors installing on Centos 7

### DISCUSSION THREAD

**Discussion by artnose** [ImageMagick/ImageMagick#2857] [General]
I'm using IM 7.0.10-27, and trying to upgrade to lastest..   Suddenly many new dependencies? 
Easy way to resolve this, shouldn't these be included in the lib.rpm if they aren't available through yum?  

Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libpng16.so.16()(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libpng16.so.16(PNG16_0)(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libIlmImf-2_2.so.22()(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libIlmThread-2_2.so.12()(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libHalf.so.12()(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libIex-2_2.so.12()(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libIexMath-2_2.so.12()(64bit)
Error: Package: ImageMagick-libs-7.0.10-43.x86_64 (/im-lib)
           Requires: libImath-2_2.so.12()(64bit)

**Comment by artnose** [comment]
And yes, I have epel installed epel-release-latest-7.noarch.rpm

**Reply by Kry1702** [reply]
Personal recommendation source installation.
Need to pay attention to compilation parameters.
